### PR TITLE
asciidoctor: add Russian translation

### PR DESCRIPTION
**Pull request by yuriskinfo**
Hi, added the Russian translation for Asciidoctor. Thanks

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ru/common/asciidoctor.md
Comment (sbrl) [inline pages.ru/common/asciidoctor.md:8]: Could you also translate the file paths please? Though of course the extensions should remain the same.

File: pages.ru/common/asciidoctor.md
Comment (EmilySeville7cfg) [inline pages.ru/common/asciidoctor.md:12]:
```suggestion
- Преобразовать данный `.adoc` файл в HTML и привязать к таблице стилей CSS:

`asciidoctor --attribute=stylesheet={{путь/до/таблицы-стилей.css}} {{путь/до/файла.adoc}}`
```

Mandatory fix (error source: original page): [broken option](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax): long option expected.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

**Review by blueskyson — APPROVED** [review]

**Comment by yuriskinfo** [comment]
I have nothing to change here, just waiting for the committers review.

**Comment by EmilySeville7cfg** [comment]
@kbdharun, can u review this PR? I have no rights to approve it and make it mergeable.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**
